.github/workflows/main.yml

```diff
@@ -1994,6 +1997,12 @@ jobs:
       - name: tomcat5
         run: ant $OPTS -f enterprise/tomcat5 test
 
+      - name: enterprise/web.jsf
+        run: .github/retry.sh ant $OPTS -f enterprise/web.jsf test
```

Comment:
` testPrettyFacesModel: org.netbeans.modules.web.jsf.metamodel.SeveralXmlModelTest` seems to be fairly unreliable.
 
It fails despite retry script
https://github.com/apache/netbeans/actions/runs/5529530202/attempts/1
https://github.com/apache/netbeans/actions/runs/5511016044/attempts/1

we should probably put it on default excludes (`enterprise/web.jsf/nbproject/project.properties`) to reduce the whack-a-mole factor.

Reply:
Added to ignore list.

Reply:
thanks!